src/EventManagement.Web/Controllers/Api/OrdersController.cs

```diff
@@ -204,5 +234,18 @@ public class OrderLineUpdateVM
 			public int Quantity { get; set; }
 			public decimal Price { get; set; }
 		}
+
+		public class UpdateOrderVM
+		{
+			public List<ProductVM> Products { get; set; } = new List<ProductVM>();
+			public int RegistrationId { get; set; }
+		}
+
+		public class ProductVM
+		{ 
+			public int Id { get; set; }
+			public int Quantity { get; set; }
+			public int? Variant { get; set; }
```

Comment:
Should this be named VariantId ? I am a simple mind you know..

Reply:
Yes, that can be done.

Reply:
I think we will do that.

Reply:
Done in 6867787.